packages/styles/scss/globals/utils/_card-content-width.scss

```diff
@@ -0,0 +1,13 @@
+//Sets the Width and Max-Width of the Card to 90% / carbon--mini-units(80)
+
+@mixin card-content-width {
```

Comment:
This is most likely going to go beyond just the card. You might just call it `content-width` because of that.